torchdynamo/variables/tensor.py

```diff
@@ -106,6 +106,11 @@ def create(cls, tx, proxy, example_value=None, nnmodule=None, **options):
                 )
         elif example_value is None or proxy.node.target is torch.manual_seed:
             return variables.ConstantVariable(None, **options)
+        elif (
+            isinstance(example_value, bool)
+            and proxy.node.target is torch.overrides.is_tensor_like
+        ):
+            return variables.ConstantVariable(example_value, **options)
```

Comment:
This is the wrong place to make it a constant.  At this point we already added a node to the FX graph, so it is too late.

You could handle this similar to `torch.is_tensor` which is here:
https://github.com/facebookresearch/torchdynamo/blob/1bc7be4de849aeb3aacee1259bfb593fce79a137/torchdynamo/variables/torch.py#L104

Reply:
L104 handles the True case.  For the False case you can add it here:
https://github.com/facebookresearch/torchdynamo/blob/1bc7be4de849aeb3aacee1259bfb593fce79a137/torchdynamo/variables/torch.py#L72